extensions/ntfy/src/ntfy.ts

```diff
@@ -0,0 +1,132 @@
+import { Toast, closeMainWindow, getPreferenceValues, getSelectedText, showToast } from '@raycast/api';
+import { showFailureToast } from '@raycast/utils';
+
+type Arguments = {
+  url?: string;
+  message?: string;
+  topic?: string;
+};
+
+type Preferences = {
+  defaultTopic: string;
+  defaultServer: string;
+  cache: boolean;
+};
+
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

```suggestion
```